test/srt/test_nightly_gsm8k_eval.py

```diff
@@ -14,12 +17,35 @@
     popen_launch_server,
 )
 
+# Define score thresholds for each model (99% of the previous score)
```

Comment:
The previous default threshold of 0.5 was too low, but due to the instability of eval, setting a single value can easily lead to failure. Therefore, we choose to set it at 99% here, which is stricter than the original 0.5.